deploy/helm/sumologic/values.yaml

```diff
@@ -208,7 +208,8 @@ fluentd:
     chunkLimitSize: "1m"
     queueChunkLimitSize: 128
     totalLimitSize: "128m"
-    retryTimeout: "1h"
+    retryMaxInterval: "1h"
```

Comment:
I have a feeling this should be lower than 1h.
Does it hurt to retry sending every 15 or 10 min?

Reply:
shouldn't too much

Reply:
I think lower than one hour is good so that once things get resolved downstream, we are not waiting the full hour to recover.